app/component/search/search.scss

```diff
@@ -23,6 +23,26 @@ $search-form-input-overlay-background-color: #EEF1F3;
   vertical-align: middle;
 }
 
+.search-form {
+  .navigate-or-input-messages {
+    color: $gray-blue;
+    border-bottom-style: dotted;
```

Comment:
We should probably use something smaller style? 
e.g. border-bottom: 1px dotted;
Or what do you think?